zilliqa/src/api/types/eth.rs

```diff
@@ -34,6 +91,9 @@ pub struct Block {
     pub size: u64,
     pub transactions: Vec<HashOrTransaction>,
     pub uncles: Vec<B256>,
+    pub qc: QuorumCertificate,
```

Comment:
Bit of a nit - Can we use `quorum_certificate` (and `aggregate_quorum_certificate`) instead of `qc` everywhere for the response fields? I don't think we should expect clients to know what `qc` means.